src/components/Releases.js

```diff
@@ -107,10 +107,13 @@ const communityReleases = [
                 modules: []
             },
             {
-                title: 'Eclipse CDT.cloud Debug Adapter',
+                title: 'Eclipse CDT Cloud Debug Adapter',
                 url: 'https://projects.eclipse.org/projects/ecd.cdt-cloud',
-                version: 'TBD',
-                modules: []
+                version: '0.0.93',
+                modules: [
+                    {modulename: 'cdt-gdb-vscode', url: 'https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode/0.0.93'},
```

Comment:
I just made a new version that pulls in all the latest and greatest. I expect it to be published overnight to https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode/0.0.95

Reply:
```suggestion
                    {modulename: 'cdt-gdb-vscode', url: 'https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode/0.0.95'},
```